community-containers/lldap/readme.md

```diff
@@ -0,0 +1,10 @@
+## Light LDAP server
+This container bundles LLDAP server and auto-configures your nextcloud instance for you.
+
+### Notes
+
```

Comment:
As discussed above, please add the commands how to add the ldap config here.

Also please add at least the following bullet point at the bottom:
`- See https://github.com/nextcloud/all-in-one/tree/main/community-containers#community-containers how to add it to the AIO stack`